js/repl.js

```diff
@@ -57,9 +58,30 @@ function compile() {
         }
         output.setValue(code, -1);
       } else if (result.type === 'error') {
+        let errorText = result.data;
+        if(!errorText.startsWith('Unexpected')){
+        errorOutput.style.display = 'block';
```

Comment:
Please use style of existing if statements -- spaces around parentheses and indentation of blocks.